demosplan/DemosPlanCoreBundle/Controller/Map/DemosPlanMapController.php

```diff
@@ -74,9 +75,11 @@ public function mapAdminAction(
             ]
         );
 
+        $procedure = $procedureService->getProcedure($procedureId);
+
         return $this->renderTemplate(
             '@DemosPlanCore/DemosPlanMap/map_admin.html.twig',
-            ['procedure' => $procedureId, 'title' => 'drawing.admin.adjustments.gis']
+            ['procedure' => $procedureId, 'master' => $procedure?->getMaster(), 'title' => 'drawing.admin.adjustments.gis']
```

Comment:
Am I right, that 'master' is a bool? If so, can we rename this to 'isMaster' ?
And would this be a good opportunity the rename 'procedure' to 'procedureId' . That would be great.

Reply:
adjusted in [ee2df92](https://github.com/demos-europe/demosplan-core/pull/3738/commits/ee2df92119435d2c1c577d758d4f1a13e4b93cd8)

Reply:
Done here [25c7ba5](https://github.com/demos-europe/demosplan-core/pull/3738/commits/25c7ba5058b51c9b31a9896923cdd97754e6a7c8)